dags/services/AWSService.py

```diff
@@ -0,0 +1,40 @@
+import boto3
+from airflow.models import Variable
+
+# AWS creds
+AWS_CREDS = {
+    "AWS_ACCESS_KEY_ID": Variable.get("TOWER_DB_ACCESS_KEY"),
+    "AWS_SECRET_ACCESS_KEY": Variable.get("TOWER_DB_SECRET_ACCESS_KEY"),
+}
+# AWS region
+AWS_REGION = "us-east-1"
+
+def initialize_aws_client(resource: str, aws_creds: dict = AWS_CREDS, aws_region: str = AWS_REGION):
+    """Initializes aws client
+
+    Args:
+        resource (str): resource type to be initialized
+        aws_creds (dict): AWS access key and secret access key dictionary
+        aws_region (str): Regoin for client to be initialized in
+    """
+    return boto3.client(
+        resource,
+        aws_access_key_id=aws_creds["AWS_ACCESS_KEY_ID"],
+        aws_secret_access_key=aws_creds["AWS_SECRET_ACCESS_KEY"],
+        region_name=aws_region,
+    )
+
+def upload_file_s3(file_path: str, bucket_name: str) -> str:
+    """Uploads file from file_path to s3 bucket_name
+
+    Args:
+        file_path (str): Path to file to be uploaded
+        bucket_name (str): Location in S3 for file to be uploaded
+    
+    Returns:
+        str: uri pointing to new uploaded file location in s3
+    """
+    s3_client = initialize_aws_client(resource="s3", aws_creds=AWS_CREDS, aws_region=AWS_REGION)
+    file_name = file_path.split("/")[-1]
```

Comment:
Functionally the same, but I think it's more readable: 

```suggestion
    file_name = os.path.basename(file_path)
```

You can also use `pathlib.Path()` to achieve the same, which would be even simpler if the function expected a `Path` object to begin with in the type hints: 

```suggestion
    file_name = Path(file_path).name
    
    # Or if it's already a Path object
    file_name = file_path.name
```

Reply:
added in latest commit